pandas/tests/indexes/multi/test_analytics.py

```diff
@@ -330,13 +330,13 @@ def test_numpy_ufuncs(idx, func):
 
     if _np_version_under1p17:
         expected_exception = AttributeError
-        msg = "'tuple' object has no attribute '{}'".format(func.__name__)
+        msg = f"'tuple' object has no attribute '{func.__name__}'"
     else:
         expected_exception = TypeError
         msg = (
-            "loop of ufunc does not support argument 0 of type tuple which"
-            " has no callable {} method"
-        ).format(func.__name__)
+            f"loop of ufunc does not support argument 0 of type tuple which"
```

Comment:
```suggestion
            "loop of ufunc does not support argument 0 of type tuple which"
```

Shouldn't need the f-prefix unless actually substituting